src/contracts/ProposalPayload.sol

```diff
@@ -1,82 +1,68 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.17;
 
-import {AaveMisc} from 'aave-address-book/AaveMisc.sol';
+import {MiscEthereum} from 'aave-address-book/MiscEthereum.sol';
 import {ProxyAdmin} from 'solidity-utils/contracts/transparent-proxy/ProxyAdmin.sol';
 import {TransparentUpgradeableProxy} from 'solidity-utils/contracts/transparent-proxy/TransparentUpgradeableProxy.sol';
 import {GenericProposal} from '../libs/GenericProposal.sol';
-import {DistributionTypes} from 'aave-stk-gov-v3/lib/DistributionTypes.sol';
-import {IAaveDistributionManager} from 'aave-stk-gov-v3/interfaces/IAaveDistributionManager.sol';
-import {Vault} from '../interfaces/Actions.sol';
-import {Addresses} from '../libs/Addresses.sol';
+import {DistributionTypes} from 'stake-token/contracts/lib/DistributionTypes.sol';
+import {IAaveDistributionManager} from 'stake-token/contracts/IAaveDistributionManager.sol';
+import {IAggregatedStakeToken} from 'stake-token/contracts/IAggregatedStakeToken.sol';
+import {AaveV3EthereumAssets} from 'aave-address-book/AaveV3Ethereum.sol';
+import {AaveV3EthereumAssets} from 'aave-address-book/AaveV3Ethereum.sol';
+import {AaveSafetyModule} from 'aave-address-book/AaveSafetyModule.sol';
 
 /**
  * @title StkABPTV2 Proposal
  * @author BGD Labs
  * @notice migrates emissions to a new stkABPT
  */
 contract ProposalPayload {
-  address public constant AAVE = Addresses.AAVE;
-  address public constant WSTETH = Addresses.WSTETH;
-  address public constant STK_ABPT_V1 = Addresses.STK_ABPT_V1;
-  Vault public constant VAULT = Vault(Addresses.BALANCER_VAULT);
+  uint256 public constant EMISSION_PER_SECOND = 4456018518518518; // same as current
```

Comment:
Just to confirm, this is fine because the emission per second is X amount of rewards given to the total staked, no matter how much is at the moment, or its units.
Right? Double-checking because in absolute numbers, the total supply units of bptv2 will for sure be different than v1

Reply:
yes it's fine